### DISCUSSION TITLE
[SCRIPT] Arrondir un chiffre dans Synapps

### DISCUSSION THREAD

**Discussion by FlorentAdell** [witsa/synapps#644] [Show and tell]
Bonjour,

J'ai eu un soucis quand j'ai développé une synapps pour un client.
Je voulais que le chiffre affiché soit arrondit car sinon ça affichait (exemple : 69.2563326)

Je voulais plutôt : 68 

J'ai utilisé une fonction dans onReadTransform sur ma variable 

![image](https://github.com/witsa/synapps/assets/46195964/510ac0a2-506b-4fe9-b861-f2eb5cfd56da)

![Screenshot_5](https://github.com/witsa/synapps/assets/46195964/beed7409-f8dd-472e-a151-225ece587b32)

![Screenshot_3](https://github.com/witsa/synapps/assets/46195964/36d95c25-4fc2-4ad8-ad0a-ac48143c1db9)

![image](https://github.com/witsa/synapps/assets/46195964/dfc24e20-fcbd-4725-891a-a2b5eda7755f)

Code : `return (Math.round(context.value));`

**Comment by WITNicolas** [comment]
Bien vu.
J'avais cela moi,
``
return parseInt(context.value).toFixed(2)
``
ou bien
``
return parseFloat(context.value).toFixed(2)
``
to.Fixed pour donner le nombre de décimale

**Reply by FlorentAdell** [reply]
Ça peut être intéressant pour certain je pense ! Parfait de l'avoir notifié !
